tests/trackingtest.py

```diff
@@ -0,0 +1,404 @@
+        worse = [match for match in self.matches if match.status == -1]
+        print("*******{} Results ******".format(self.id))
+        print(
+            "matches {}\tmismatches {}\tunmatched {}".format(
+                len(matched), len(unmatched), len(self.unmatched_tracks)
+            )
+        )
+        print("*******Tracking******")
+        print("same {} better {}\t worse {}".format(len(same), len(better), len(worse)))
+        if len(self.unmatched_tests) > 0:
+            print("unmatched tests {}\t ".format(len(self.unmatched_tests)))
+        summary = {
+            "classify": {"correct": len(matched), "incorrect": len(unmatched)},
+            "tracking": {
+                "better": len(better),
+                "same": len(same),
+                "worse": len(worse),
+                "unmatched_tracks": len(self.unmatched_tracks),
+                "unmatched_tests": len(self.unmatched_tests),
+            },
+        }
+        return summary
+
+    def write_results(self, f):
+        f.write("{}{}{}\n".format("-" * 10, "Recording", "-" * 90))
+        f.write("Recordings[{}] {}\n\n".format(self.id, self.filename))
+        for match in self.matches:
+            match.write_results(f)
+
+        if len(self.unmatched_tracks) > 0:
+            f.write("Unmatched Tracks\n")
+        for (what, track) in self.unmatched_tracks:
+            f.write(
+                "{} - [{}s]Start-End {} - {}\n".format(
+                    what,
+                    round(track.end_s - track.start_s, 1),
+                    round(track.start_s, 1),
+                    round(track.end_s, 1),
+                )
+            )
+        f.write("\n")
+
+        if len(self.unmatched_tests) > 0:
+            f.write("Unmatched Tests\n")
+        for expected in self.unmatched_tests:
+            f.write(
+                "{} - Opt[{}s] Start-End {} - {}, Expected[{}s] {} - {}\n".format(
+                    expected.tag,
+                    round(expected.opt_end - expected.opt_start, 1),
+                    expected.opt_start,
+                    expected.opt_end,
+                    round(expected.end - expected.start, 1),
+                    expected.start,
+                    expected.end,
+                )
+            )
+        f.write("\n")
+
+
+class Match:
+    def __init__(self, test_track, track, tag):
+        expected_length = test_track.opt_end - test_track.opt_start
+        self.length_diff = round(expected_length - (track.end_s - track.start_s), 2)
+        self.start_diff_s = round(test_track.start - track.start_s, 2)
+        self.end_diff_s = round(test_track.end - track.end_s, 2)
+        self.opt_start_diff_s = round(test_track.opt_start - track.start_s, 2)
+        self.opt_end_diff_s = round(test_track.opt_end - track.end_s, 2)
+        self.error = round(abs(self.opt_start_diff_s) + abs(self.opt_end_diff_s), 1)
+
+        if self.error <= test_track.calc_error():
+            self.status = 1
```

Comment:
Status should be an enum not a **magic number** - 1.  More information https://docs.python.org/3/library/enum.html